Include/pyport.h

```diff
@@ -505,13 +505,16 @@ extern "C" {
 /* Py_DEPRECATED(version)
  * Declare a variable, type, or function deprecated.
```

Comment:
Please add: "The macro must be placed before the symbol name."

Reply:
Before the declaration.

Reply:
Done.